Modules/_testinternalcapi.c

```diff
@@ -957,6 +957,16 @@ iframe_getlasti(PyObject *self, PyObject *frame)
     return PyLong_FromLong(PyUnstable_InterpreterFrame_GetLasti(f));
 }
 
+static PyObject *
+get_co_framesize(PyObject *self, PyObject *arg) {
```

Comment:
I really like how comprehensive this test is. But I don't think we shouldn't test for such implementation details. We should just test the bytecode trace is correct and that's the limit.

Also, `_CHECK_STACK_SPACE` shouldn't affect frame size. It's a check after all.

Reply:
I'm going to remove the assertions for exact framesizes, but I'd still like to keep this function. It's useful to help determine that `_CHECK_STACK_SPACE_OPERAND` is assigned the correct operand.

To be clear, I'm not worried about my changes affecting the framesizes themselves, just that the calculated total stack space required is accurate.